configs/dc2.yaml

```diff
@@ -0,0 +1,51 @@
+facility_name: Rubin-LSST
+obs_collection: LSST.DC2
+collections: ["2.2i/runs/DP0.1"]
+use_butler_uri: False
+dataset_types:
+  raw:
+    dataproduct_type: image
+    dataproduct_subtype: lsst.raw
+    calib_level: 1
+    obs_id_fmt: "{records[exposure].obs_id}"
+    o_ucd: phot.count
+    access_format: image/fits
+    datalink_url_fmt: "http://datalink.org/{obs_id}"
+  calexp:
+    dataproduct_type: image
+    dataproduct_subtype: lsst.calexp
+    calib_level: 2
+    obs_id_fmt: "{records[visit].name}"
+    o_ucd: phot.count
+    access_format: image/fits
+    datalink_url_fmt: "http://datalink.org/{obs_id}"
+  deepCoadd_calexp:
+    dataproduct_type: image
+    dataproduct_subtype: lsst.coadd
+    calib_level: 3
+    obs_id_fmt: "{skymap}-{tract}-{patch}"
+    o_ucd: phot.count
```

Comment:
The coadds presumably aren't in counts still.

Reply:
Agreed.  I'll get the correct value.

Reply:
I am clearly not qualified to make a correct guess, this was made as an example. I'm going to depend on someone to make a correct configuration later.